storage/rocksdb/ha_rocksdb.cc

```diff
@@ -3367,8 +3347,52 @@ class Rdb_snapshot_status : public Rdb_tx_list_walker {
     m_data += "----------LATEST DETECTED DEADLOCKS----------\n";
 
     for (auto path_entry : dlock_buffer) {
-      m_data += get_dlock_path_info(path_entry);
+      std::string path_data;
+      const auto dl_info = get_dl_path_trx_info(path_entry);
+      if (path_entry.limit_exceeded) {
+        path_data += "\n-------DEADLOCK EXCEEDED MAX DEPTH-------\n";
+      } else {
+        path_data += "\n*** DEADLOCK PATH\n"
+                     "=========================================\n";
+        for (auto it = dl_info.path.begin(); it != dl_info.path.end(); it++) {
+          const auto trx_info = *it;
+          path_data += format_string(
+              "TRANSACTIONID: %u\n"
```

Comment:
This isn't from your diff, but I think "TRANSACTION ID" would be more consistent.